postgres/datadog_checks/postgres/data/conf.yaml.example

```diff
@@ -143,9 +143,11 @@ instances:
     ## * f = foreign table
     ## * p = partitioned table
     ##
+    ## When configuring relations, you must provide either `relation_name` or `relation_regex` in every entry, even
+    ## if you are primarily using `schemas` or `relkind` arrays to specify which relations to track.
+    ##
     ## Note: For compatibility reasons you can also use the following syntax to track relations metrics by specifying
     ## the list of table names. All schemas are included and regex are not supported.
-    ## relations:
     #
```

Comment:
You should not edit this file directly. Instead, you'll need to edit [this one](https://github.com/DataDog/integrations-core/blob/master/postgres/assets/configuration/spec.yaml) and run the `ddev validate config postgres -s` command to update the example file.

Reply:
Thanks for the correction and for your patience. I've made the edits to the correct file as you requested.